joblib/test/test_memory.py

```diff
@@ -640,6 +640,67 @@ def test_memory_file_modification():
     nose.tools.assert_equal(my_stdout.getvalue(), '1\n2\nReloading\nx=1\n')
 
 
+def test_memory_arg_lambda():
+    " Test memory with a lambda argument."
```

Comment:
I tend to never use docstring in test functions and use comments instead. The reason [why](https://github.com/sphinx-gallery/sphinx-gallery/pull/85#discussion_r49304536).